files/private-chef-cookbooks/private-chef/libraries/helper.rb

```diff
@@ -114,6 +114,9 @@ def self.gen_certificate
     cert_uuid = UUIDTools::UUID.random_create
     common_name = "URI:http://opscode.com/GUIDS/#{cert_uuid}"
     info = [["C", "US"], ["ST", "Washington"], ["L", "Seattle"], ["O", "Opscode, Inc."], ["OU", "Certificate Service"], ["CN", common_name]]
+
+    # TODO: we should no longer be using certificates. we should see if anyone is still
+    #       using thing and remove if not.
```

Comment:
Well the code is being used. I'm not sure if the cert is being used anywhere. It isn't to my knowledge.